.github/workflows/test-code-code2parquet.yml

```diff
@@ -0,0 +1,95 @@
+#
+# DO NOT EDIT THIS: it is generated from test-transform.template,  Edit there and run make to change these files
+#
+name: Test - transforms/code/code2parquet
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - "dev"
+            - "releases/**"
+        tags:
+            - "*"
+        paths:
+            - "transforms/code/code2parquet/**"
+            - "data-processing-lib/**"
+    pull_request:
+        branches:
+            - "dev"
+            - "releases/**"
+        paths:
+            - "transforms/code/code2parquet/**"
+            - "data-processing-lib/**"
+
+jobs:
+    check_if_push_image:
```

Comment:
typo check_if_push_image -> check_if_push_images

Reply:
thank you. that fixed some odd behavior i was seeing.